src/redux/tests/lessonsReducer.test.js

```diff
@@ -0,0 +1,114 @@
+import { fetchLessons } from "../lessons/asyncActions";
+import lessonsReducer from "../lessons/slice";
+
+const initialState = {
+    status: 'loading',
+    lessons: [],
+    courseInfo: null,
+}
+
+const lessons = {
```

Comment:
I think, it would be better to import this data mock from some module rather than defining it in the test code